galley/pkg/server/components/processing.go

```diff
@@ -59,6 +59,8 @@ const versionMetadataKey = "config.source.version"
 type Processing struct {
 	args *settings.Args
 
+	excludedResourceKinds []string
```

Comment:
Please don't change processing. This is the mainline code it is harder to argue that we're not changing any production code if you're changing this file.

Reply:
Discussed offline. We don't actually need these changes right now, so removing them.